### ISSUE TITLE
Adicionar um unrar dentro do próprio add-on e definir o idioma da legenda com as tags IETF

### ISSUE THREAD

**Issue by edjalmobf** [gfjardim/service.subtitles.legendastv#28]
Vi que vc consertou o problema de extração das legendas no Kodi 18 usando o add-on `RAR archive support`. Mas creio que seria muito melhor se adiciona-se um extrator dentro do próprio add-on, pois da maneira que está confunde bastante quem simplesmente instala ele via o repositório oficial do Kodi.

(eu estava tendo esse mesmo problema no legendastv com o [Subliminal](https://github.com/dtroncy/docker-rpi-subliminal). Falando com o desenvolvedor ele [adicionou o `unrar`](https://github.com/dtroncy/docker-rpi-subliminal/commit/d9da6645f2b8211637fdcad37cd2ddd3b740f54d) dentro do container e isso resolveu o problema!)

Outra coisa é: teria como adicionar um `.pt-BR` no final do nome do arquivo da legenda? 
Pq dessa forma o Kodi entende que a legenda está no idioma português do Brasil e é carregada como padrão junto do vídeo, caso a configuração esteja definida assim. Pelo que entendi o Kodi entende as [tags IETF](https://gist.github.com/traysr/2001377).

É isso! Obrigado pelo trabalho com o add-on e abraço! :)

**Comment by andrebrait** [comment]
Acredito que posso tentar olhar isso. Seria melhor adicionar isso ao nome da legenda, de fato.

**Comment by edjalmobf** [comment]
> Acredito que posso tentar olhar isso. Seria melhor adicionar isso ao nome da legenda, de fato.

Essa parte do rar já foi praticamente solucionada pelo @henricos aqui https://github.com/henricos/service.subtitles.legendastv (digo praticamente pq as vezes ainda acontecem erros na hora de extrair)
Agora a parte de adicionar o .pt-BT a legenda não.
Seria muito bom se pudessem portar o addon pro python 3, já q na próxima versão do Kodi o suporte ao python 2 acaba.

**Comment by andrebrait** [comment]
Sim, a parte do RAR já está funcionando. Eu estava falando de adicionar a língua ao nome da legenda mesmo.
Estou submetendo a versão mais atual nos repositórios do Kodi 18 (Leia) e já tenho algumas mudanças pra fazer por conta de mudanças no Kodi (a pedido do reviewer do PR).

Depois que isso tiver sido feito, começo a portar para Python 3. Tenho que ver se o Kodi 18 já suporta a versão 3. 
Parece que os repositórios para o Kodi 19 serão separados, de qualquer forma, então vai ser necessário realizar algumas mudanças de qualquer forma.

**Comment by edjalmobf** [comment]
@andrebrait na verdade essa parte de adicionar a extensão do idioma no nome do arquivo da legenda já esta resolvido tbm. Eu fiquei procurando no forum do Kodi até q encontrei uma maneira, veja aqui como fazer funcionar: https://medium.com/@edjalmo/fazendo-o-kodi-mostrar-o-idioma-das-legendas-em-portugu%C3%AAs-do-brasil-e-adicionando-animes-sem-48cf9014b492
Desse jeito o Kodi mostra `Portuguese - br` como idioma.
A partir da versão 19 do Kodi isso já vai ser habilitado por padrão, sem precisar dessa gambiarra.
Se posso dar uma outra sugestão é colocar o ˜BR˜ em maiúsculas, o addon faz assim `pt-br`, acho mais bonito `pt-BR` (fincando `Portuguese - BR`) :)

Em alguns filmes com legendas embutidas ele mostra `Portuguese Brazil`, sendo muito mais elegante, mas não sei se da pra fazer isso só modificando a extensão do nome da legenda.
